### ISSUE TITLE
AUDIO: see if TriggerAttackRelease() can work

### ISSUE THREAD

**Issue by Whitevinyl** [BlokDust/BlokDust#295]
Current function calls TriggerAttack() followed by TriggerRelease().

I'm thinking the cancelScheduledValues line in the 2nd function negates the first, so the attack doesn't take place.
Currently I think it's only the Granular that uses this, and I've temporarily replaced it's usage with a setTimeout.

**Comment by lukephills** [comment]
Yeah you're right. I removed the 2nd cancelScheduledValues call and it's working now.
https://github.com/BlokDust/Tone.js/commit/54d2af75905ea0222b3a12afe525b31ff3c80bfb

I've found a bug with the Granular which could possibly be fixed when you switch back to triggerAttackRelease but not sure. You can recreate it by connecting a Granular to a Power and then load a new sample. You'll get InvalidStateErrors and the Granular can't be triggered again.

**Comment by Whitevinyl** [comment]
Ah Ok, I've fixed the Granular bug.
Yeah the triggerAttackRelease is calling now, I've put it back to a timeout for the time being though as it sounds much smoother, it's difficult to tell with the short times, but I'm not convinced the AttackRelease is ramping - will spend some more time tinkering/testing when I can.
